### ISSUE TITLE
Your website is unsafe

### ISSUE THREAD

**Issue by ITR13** [S0FTY/goodhub#1]
As you seem to have noticed, there are some glaring errors in the code of your website. I checked your website out because you wanted people to pay you to not pirate Besiege, but hopefully, nobody did that.

First of all, all your forms are susceptible to MySQL-injections, For Instance, you can log into a random user by having "' OR 1=1 LIMIT 1#" into the username field and putting anything in the password field, or as any user by putting "[username]' #", or any user with a specific ID by writing "' AND `id`='[id]'". 

But since you also check cookies for a "user" and "password" cookie, and then just set the ID and e-mail to that without checking if it's actually correct, it's possible to just edit your own cookies to get in

In addition, since the song-names are displayed on the home-page, XSS scripts can be put in the home page to attack any logged in user. 

I didn't test this, but I assume it's also possible to find out the passwords of all the users too, using a variation of the previously mentioned SQL-injection, as you don't encrypt the passwords

So to sum up, the website is unsafe for users, and not something you should try to earn money on

**Comment by S0FTY** [comment]
Scrapped the project after using pdo. FYI I did test for SQL Injections while using Mysqli. It didn't work. Also I didn't pirate besieged. I ended up buying it. I started working on this site long before I wanted besieged.

**Comment by S0FTY** [comment]
And no it isn't because the passwords were hashed.

**Comment by RedDogCode** [comment]
Agreed, partly why I took it down after I switched to PDO. Ended up getting
my own copy of the game.
